app/controllers/content_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ContentController < ApplicationController
+  def edit
+    @edition = Edition.find_current(document: params[:document])
+    assert_edition_state(@edition, &:editable?)
+    @revision = @edition.revision
+  end
+
+  def update
+    result = Editions::UpdateInteractor.call(params: params, user: current_user)
```

Comment:
Should this be under the `Content` namespace?

Reply:
yep, should have been, just fixing it now :)